pkg/proxy/ipvs/proxier.go

```diff
@@ -899,6 +899,23 @@ func (proxier *Proxier) OnEndpointSlicesSynced() {
 // EntryInvalidErr indicates if an ipset entry is invalid or not
 const EntryInvalidErr = "error adding entry %s to ipset %s"
 
+func getLocalAddrs() (map[string]bool, error) {
```

Comment:
Make this a `map[string]struct{}` since we only check for existence

Reply:
git it